src/Gui/DlgUnitsCalculatorImp.cpp

```diff
@@ -58,14 +58,13 @@ DlgUnitsCalculator::DlgUnitsCalculator( QWidget* parent, Qt::WindowFlags fl )
     }
 
     connect(ui->ValueInput, SIGNAL(valueChanged(Base::Quantity)), this, SLOT(valueChanged(Base::Quantity)));
```

Comment:
@wwmayer , for this connection I found no solution to modernize. Do you know a way?